MODULE.bazel

```diff
@@ -1,2 +1,22 @@
 # TODO: migrate all dependencies from WORKSPACE to MODULE.bazel
 # https://github.com/protocolbuffers/protobuf/issues/14313
+module(
+    name = "protobuf",
+    compatibility_level = 1,
+    version = "23.1",
```

Comment:
Could we bump this up to 27.0 to match protobuf_version.bzl?  Or ideally hook it up to that file if it's possible

Reply:
Sadly you cannot have load statements in MODULE.bazel so we can't reuse the constant. I copied the value.

Note in most rulesets we just leave the version in code as a placeholder, to avoid having to create "version bump commits" - I can show you more about it if you're curious.

Reply:
Yea I'd be curious about possible better solutions, but we have *many* places we need to bump every release.  Our tooling handles this automatically